nist/int.go

```diff
@@ -416,3 +434,14 @@ func (i *Int) HideDecode(buf []byte) {
 	i.V.SetBytes(buf)
 	i.V.Mod(&i.V, i.M)
 }
+
+// SetEndianess will set the endianness of the value to end.
+func (i *Int) SetEndianness(end binary.ByteOrder) {
```

Comment:
Since I'm going for a Public field as suggested above, we can even remove that method